### DISCUSSION TITLE
New task lists in project boards and issue search

### DISCUSSION THREAD

**Discussion by leighmcculloch** [community/community#4276] [Projects and Issues]
I'm loving the new task lists features that showed up today. It would be really helpful if there were two additional places that task lists surfaced:

 - [ ] Project Boards – It'd be super helpful if there was a way to have a project board filtered to show all the sub-tasks of a task. It would help quickly zoom in / focus on the tasks related to some larger piece of work.
 - [ ] Issue Search – It'd also be helpful, although less critical I think, if there was a way to search for issues that are tracked by another issue.

Thanks!

**Comment by cristiklein** [comment]
- [ ] Searching for Issues that are *not* tracked by another issue. Useful when grooming the backlog.

**Comment by itcarroll** [comment]
In the issue search, we could have `tasks:n` analogous to `comments:n`, and `is:tracked`. How about `tasks:>90%` for all PRs that are almost ready to merge!

**Reply by nexushoratio** [reply]
Is `is:tracked` really a thing?  I could find no documentation on it, and not sure if it does anything in *issue search*

**Comment by nexushoratio** [comment]
Partial duplicate:
- #7475
